tools/azure_no_deprecated.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index dfb441e..af3dd36 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
```

Comment:
does this file need to be here?

Reply:
@GuanLuo , could you please advise on it?

Reply:
This is patch to Azure's CMake to ignore deprecated warning, otherwise it will be reported as error and fail the build unless we address the Azure upgrade